File: NLHDH/November-15/file.java
import java.io.File;

public class file {
  public static void main(String[ ] args) {
    File x = new File("C:\\Users\\dung5\\OneDrive\\HOMEWORK\\NLHDH");

    System.out.println("File name: " + x.getName());
    System.out.println("File parent: " + x.getParent());
    System.out.println("File path: " + x.getPath());


    if(x.exists()) {
     System.out.println(x.getName() +  " exists!");
    }
    else {
     System.out.println("The file does not exist");
    }
  }
}

File: NLHDH/November-15/file.cpp
#include <iostream>
#include <string>
#include <fstream>
#include <experimental/filesystem>

namespace fs = std::experimental::filesystem;

int main()
{
  fs::path path("C:\\Users\\dung5\\OneDrive\\HOMEWORK\\NLHDH");
  for (auto& p : fs::directory_iterator(path))
    std::cout << p << '\n';
  
}